presto-main/src/test/java/io/prestosql/sql/planner/optimizations/TestWindow.java

```diff
@@ -117,8 +119,14 @@ public void testWindow()
     @Test
     public void testWindowAfterJoin()
     {
+        Session disableCbo = Session.builder(getQueryRunner().getDefaultSession())
+                .setSystemProperty(JOIN_DISTRIBUTION_TYPE, PARTITIONED.name())
+                .setSystemProperty(JOIN_REORDERING_STRATEGY, ELIMINATE_CROSS_JOINS.name())
```

Comment:
For this test, it seems to make sense to use JOIN_REORDERING_STRATEGY=NONE.
Then the variable: disableCbo -> noJoinReordering.